Kickstarter-iOS/Views/CommentComposerView.swift

```diff
@@ -0,0 +1,140 @@
+import Library
+import Prelude
+import UIKit
+
+private enum Layout {
+  enum Border {
+    static let height: CGFloat = 1.0
+  }
+
+  enum Avatar {
+    static let diameter: CGFloat = 44.0
+    static let margin: CGFloat = 18.0
```

Comment:
In the past we have had a concept of a 6pt grid for our designs, this allowed us to specify sizes using `Styles.grid(3)` for example to get value like this of `18`. I'm just noting this here because I'm not sure if these current designs are also using this grid, we might want to try to get the closest grid sizes to the design 🤔 My concern is that most of our current UI uses the `Styles.grid()` function and using the pt sizes directly like this would be a deviation.

cc @singhhari what do you think?

Reply:
Yeah, I also contemplated what approach to use because the `Styles.grid()` system has its units in multiples of 6 but the Figma design doesn't follow that convention.

Reply:
Yeah I'm just wondering if it wouldn't be sufficient to get it _close enough_ 🤔